packages/framework/src/Component/Paginator/QueryPaginator.php

```diff
@@ -99,10 +99,9 @@ protected function getTotalNativeQuery(QueryBuilder $queryBuilder)
             $parametersAssoc[$parameter->getName()] = $parameter->getValue();
         }
 
-        list(, $flatenedParameters) = SQLParserUtils::expandListParameters(
+        $flattenedParameters = SqlParametersFlattener::flattenArrayParameters(
```

Comment:
🤔 is this native query necessary at all? 
this whole implementation is quite uncommon. Can we discuss it please?

Reply:
ok, let's discuss it :slightly_smiling_face: